next/components/forms/info-components/Tooltip/Tooltip.tsx

```diff
@@ -1,8 +1,9 @@
 import HelpIcon from '@assets/images/forms/icon-help.svg'
+// import Button from '@components/forms/simple-components/Button'
```

Comment:
Please don't forget to clean up unused imports :)

Reply:
yes, about that TS errors. I forgot to remove imported component library in styleguide. PR in component-library with code required for it is not merged yet so we cant really import it, neither can we use ContextProvider in any place for now. It was tested locally with yalc. As for other issues, I'm looking trough it right now.